src/guide/receive_onchain.md

```diff
@@ -373,5 +373,35 @@ To calculate the fees for a channel being opened by the LSP:
 </section>
 </custom-tabs>
 
+# SwapInfo fields
```

Comment:
IMO this entire section isn't needed anymore. You already described the fields in the rustdoc.